.github/pull_request_template.md

```diff
@@ -1,15 +1,27 @@
-# Proposed Changes
+## Description (Proposed Changes)
+(Please try to mention in bullet points.)
 
--
 -
 -
 
-Related Issues (if applicable)
+## Link to the issue : 
 
--
+(Add link of the issue you have proposed changes to)
+
+- Link :
+
+## Tests
+
+I added the following tests (mention the file name as well): 
```

Comment:
I would word this differently, after all the file names would be visible in the diff. Something like "Please make sure to add tests when implementing new features"?